bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -216,6 +216,7 @@ spec:
                                   x-kubernetes-int-or-string: true
```

Comment:
Don't  we need `nullable: true` on line number 209 as well as on line number 353.